apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -334,6 +355,35 @@ export default class WorkshopForm extends React.Component {
     return this.state.course && window.dashboard.workshop.SUBJECTS[this.state.course];
   }
 
+  renderRegionalPartnerSelect() {
+    return (
+      <FormGroup>
+        <ControlLabel>
+          Regional Partner
+        </ControlLabel>
+        <FormControl
+          componentClass="select"
+          name="regional_partner_id"
+          onChange={this.handleFieldChange}
+          style={this.getInputStyle()}
+          value={this.state.regional_partner_id || ''}
+          disabled={this.props.readOnly}
+        >
+          <option value=""/>
```

Comment:
Should this null option maybe say "None", or should the control have placeholder text?